src/ZGM-17_Achilles_Blufor.Altis/initPlayerLocal.sqf

```diff
@@ -33,4 +43,7 @@ if (_is_curator) then
 	};
 	execVM "scripts\R3F_LOG\init.sqf";
 };
-
+// initalise diary record for rulesinit
+if (hasInterface) then {
```

Comment:
`hasInterface` shouldn't be required as it's `initPlayerLocal.sqf` so they will all be player machines with an interface.

Reply:
Would it suffice just having `execVM "scripts\rules.sqf";` alone then?

Reply:
Yes, it should.